src/main/java/ca/ciccc/silverBullet/SilverBulletApp.java

```diff
@@ -1,9 +1,11 @@
 package ca.ciccc.silverBullet;
 
+import FileInput.FileInput;
```

Comment:
Can you include the word Media into this FileInput class?

Reply:
And please change its package as well

Reply:
then they have to extend fileinput class to use the sound.